mojo/dart/packages/mojo/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.4.17
+
+  - 48 changes: https://github.com/domokit/mojo/compare/2829804...1484e83
+
+  MOJO_SDK: 1484e831758d05c7af0f27c199d24e3bbdc6383f
```

Comment:
This is really a mojo commit, not mojo_sdk, correct?

Do you think you could put this commit number in a separate file that's more easily readable by tools (like engine.version)? It would make it easier for us to sync all of our dependencies (which include flutter, mojo, and mojo_sdk).